tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -74,6 +74,11 @@
     'tensor_split', 'swapdims', 'swapaxes'
 })
 
+# These are not included above, but shares storage regardless
+OTHER_VIEW_FUNCTIONS = {
```

Comment:
Maybe UNSAFE_VIEW_FUNCTIONS? Other is not very descriptive.

Reply:
What's the link between this and DONT_ENFORCE_SAME_TENSOR_IMPL_OR_STORAGE ? Do we really need both of them?

Reply:
Oh good catch, there is a difference - I've changed it so that we actually check for this case. Basically we should consider `_unsafe_view` still a view for our purposes here. Its just they they weren't included in the list above because its not a view for the purposes of `gen_inplace_or_view`, i.e., we don't want to call `as_view` on its output.